base/linalg/arnoldi.jl

```diff
@@ -363,6 +364,36 @@ iterations derived from [`eigs`](@ref).
 # Example
 
 ```jldoctest
+julia> A = spdiagm(1:5);
+
+julia> Z = svds(A, nsv = 2)[1];
+
+julia> Z.U #or Z[:U]
+5×2 Array{Float64,2}:
+ -2.35514e-16   2.35514e-16
+ -3.14018e-16  -3.92523e-17
+ -7.85046e-16  -5.88785e-17
+  7.85046e-16   1.0
+  1.0           5.49532e-16
+
+julia> Z.S #or Z[:S]
+2-element Array{Float64,1}:
+ 5.0
+ 4.0
+
+julia> Z.Vt #or Z[:Vt]
+5×2 Array{Float64,2}:
+ -2.35514e-16  1.57009e-16
+  0.0          0.0
+ -7.85046e-16  5.88785e-17
+  7.06542e-16  1.0
+  1.0          3.92523e-16
+
+julia> Z[:V]
+2×5 Array{Float64,2}:
+ -2.35514e-16  0.0  -7.85046e-16  7.06542e-16  1.0
+  1.57009e-16  0.0   5.88785e-17  1.0          3.92523e-16
+
 julia> A = spdiagm(1:4);
```

Comment:
This example seem a bit superfluous when the exact same example is on the lines above.